See IZ#12442. Error message:
Checking DLL ../unxlngi4.pro/lib/check_libucpchelp1.so ...: ERROR: ../unxlngi4.pro/lib/check_libucpchelp1.so: undefined symbol: _ZT
IN3ucb25ContentProviderImplHelperE

Index: oo_cvs/xmlhelp/source/cxxhelp/provider/makefile.mk
===================================================================
RCS file: /cvs/util/xmlhelp/source/cxxhelp/provider/makefile.mk,v
retrieving revision 1.11
retrieving revision 1.11.18.1
diff -u -3 -p -r1.11 -r1.11.18.1
--- oo_cvs/xmlhelp/source/cxxhelp/provider/makefile.mk	9 Sep 2002 11:15:32 -0000	1.11
+++ oo_cvs/xmlhelp/source/cxxhelp/provider/makefile.mk	7 Apr 2003 12:13:15 -0000	1.11.18.1
@@ -78,11 +78,6 @@ CFLAGS +=  -DHAVE_EXPAT_H
 .IF "$(GUI)"=="WNT"
 CFLAGS+=-GR
 .ENDIF
-
-.IF "$(COM)"=="GCC"
-CFLAGSCXX+=-frtti
-.ENDIF
-
 
 # --- General -----------------------------------------------------
 
Index: oo_cvs/xmlhelp/source/cxxhelp/qe/makefile.mk
===================================================================
RCS file: /cvs/util/xmlhelp/source/cxxhelp/qe/makefile.mk,v
retrieving revision 1.6
retrieving revision 1.6.18.1
diff -u -3 -p -r1.6 -r1.6.18.1
--- oo_cvs/xmlhelp/source/cxxhelp/qe/makefile.mk	9 Sep 2002 11:14:40 -0000	1.6
+++ oo_cvs/xmlhelp/source/cxxhelp/qe/makefile.mk	7 Apr 2003 12:13:20 -0000	1.6.18.1
@@ -74,10 +74,6 @@ ENABLE_EXCEPTIONS=TRUE
 
 .IF "$(GUI)"=="WNT"
 CFLAGS+=-GR
-.ENDIF
-
-.IF "$(COM)"=="GCC"
-CFLAGSCXX+=-frtti
 .ENDIF
 
 SLOFILES=\
Index: oo_cvs/xmlhelp/source/cxxhelp/util/makefile.mk
===================================================================
RCS file: /cvs/util/xmlhelp/source/cxxhelp/util/makefile.mk,v
retrieving revision 1.6
retrieving revision 1.6.18.1
diff -u -3 -p -r1.6 -r1.6.18.1
--- oo_cvs/xmlhelp/source/cxxhelp/util/makefile.mk	9 Sep 2002 11:12:13 -0000	1.6
+++ oo_cvs/xmlhelp/source/cxxhelp/util/makefile.mk	7 Apr 2003 12:13:25 -0000	1.6.18.1
@@ -74,9 +74,6 @@ ENABLE_EXCEPTIONS=TRUE
 
 .IF "$(GUI)"=="WNT"
 CFLAGS+=-GR
-.ENDIF
-.IF "$(COM)"=="GCC"
-CFLAGSCXX+=-frtti
 .ENDIF
 
 SLOFILES=\
Index: oo_cvs/xmlhelp/util/makefile.mk
===================================================================
RCS file: /cvs/util/xmlhelp/util/makefile.mk,v
retrieving revision 1.3.20.1
retrieving revision 1.3.20.1.4.1
diff -u -3 -p -r1.3.20.1 -r1.3.20.1.4.1
--- oo_cvs/xmlhelp/util/makefile.mk	20 Feb 2003 17:30:05 -0000	1.3.20.1
+++ oo_cvs/xmlhelp/util/makefile.mk	7 Apr 2003 12:13:28 -0000	1.3.20.1.4.1
@@ -78,9 +78,6 @@ NO_BSYMBOLIC=TRUE
 
 .IF "$(GUI)"=="WNT"
 CFLAGS+=-GR
-.ENDIF
-.IF "$(COM)"=="GCC"
-CFLAGSCXX+=-frtti
 .ENDIF
 
 .IF "$(SABLOT3RDLIB)"==""
